docs/CONFIGURING_GLOBAL_HPKE_KEYS.md

```diff
@@ -0,0 +1,98 @@
+# Global HPKE Keypairs
+Janus has support for HPKE key advertisement on `/hpke_configs` that are not
+tied to tasks. This is mainly to support the Taskprov extension, but we are
+also [considering having this replace per-task keys entirely][1].
+
+If a global key is configured, requests to `/hpke_configs` that don't provide
+the task ID will provide the list of active global HPKE keys.
+
+This document describes the operational overhead of managing these keys.
+
+[1]: https://github.com/divviup/janus/issues/1641
+
+## Lifecycle
+A global keypair has three states:
+- `pending`: The key is in the database, but is not being advertised to clients.
+- `active`: The key is used by clients, and clients should use it to encrypt
+  reports.
```

Comment:
To align with the other two bullets, plus we can't really say what clients are or are not doing:
```suggestion
- `active`: The key is being advertised to clients, and clients should use it to encrypt
  reports.
```